### PR TITLE
Psimao pla 54

### PR DESCRIPTION
**Pull request by paulosimao-ardanlabs**
fixing name for bg-to-gcs

### CODE REVIEW

**Review by AnthonyArdronUW — CHANGES_REQUESTED** [review]
The .docs folder will also need to change to match the new format

**Comment by paulosimao-ardanlabs** [comment]
> The .docs folder will also need to change to match the new format. https://github.com/utilitywarehouse/data-infra-mono/tree/psimao-pla-54/.doc

Fixed.

File: .github/workflows/di-bq-to-gcs.yml [hunk 1]
```diff
@@ -1,13 +1,13 @@
-name: datainfra-bqtogcs-connector
+name: di-bq-to-gcs
```
Comment (catalin-ilea) [inline .github/workflows/di-bq-to-gcs.yml:1]: should's this be di-bq-source to match the readme?

Comment (paulosimao-ardanlabs) [reply .github/workflows/di-bq-to-gcs.yml:1]: indeed it should.

**Review by iainmillarUW — APPROVED** [review]

**Comment by paulosimao-ardanlabs** [comment]
> The .docs folder will also need to change to match the new format. https://github.com/utilitywarehouse/data-infra-mono/tree/psimao-pla-54/.doc

This has been addressed.

**Review by catalin-ilea — APPROVED** [review]

**Review by AnthonyArdronUW — APPROVED** [review]

**Pull request merged**